### PR TITLE
Simplify Main Entry Point

### PR DESCRIPTION
**Pull request by bh2smith**
No logical changes have been made here.

Except now there is a new function `def dune_to_postgres(env: Env, job: DuneToLocalJob) -> None:`

which is called from the main entry point.

I am not terribly fond of the new file name `dune_to_local/main.py` - open to suggestions.

### CODE REVIEW

File: src/dune_to_local/main.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+def save_to_postgres(
+    engine: sqlalchemy.engine.Engine, table_name: str, df: DataFrame, dtypes: DataTypes
+) -> None:
+    df.to_sql(table_name, engine, if_exists="replace", index=False, dtype=dtypes)
+    log.info("Data saved to %s successfully!", table_name)
+
+
+def dune_to_postgres(env: Env, job: DuneToLocalJob) -> None:
```
Comment (bh2smith) [inline src/dune_to_local/main.py]: This function is "new" but really just encapsulates what was previously in main.

**Comment by mooster531** [comment]
I'd call it `dune_to_postgres/main.py` and call it a day.

**Comment by bh2smith** [comment]
I think I will go with `dune_to_local/postgres.py`

**Comment by bh2smith** [comment]
All ready for your review.

**Review by mooster531 — APPROVED** [review]
:rocket: :moon:

**Pull request merged**